dev/io.openliberty.jakarta.pages.tld.3.0/bnd.bnd

```diff
@@ -13,7 +13,7 @@ bVersion=1.0
 -nouses=true
 
 Bundle-SymbolicName: io.openliberty.jakarta.pages.tld.3.0
-Bundle-Description: Jakarta Standard Tag Library, version 2.0
+Bundle-Description: Jakarta Standard Tag Library, version 3.0
```

Comment:
This is actually version 2.0, but the bundle name actually refers to 3.0 version in pages 3.0, not the underlying Tags technology.  I don't know why we did this. I think we should rename to bundle now that we have Jakarta Tags 3.0 out.

Reply:
So I should leave it be 2.0?

Reply:
I put it back to 2.0